web/src/interactives/covid-dashboard/countrygraph.js

```diff
@@ -1,9 +1,10 @@
-import React, {Component} from "react";
-import {fetchHistData} from "./data/dataservice";
-import {Line} from "react-chartjs-2";
+import React, { Component } from "react";
+import { fetchHistData } from "./data/dataservice";
+import { Line } from "react-chartjs-2";
```

Comment:
More for the future: at some point, do we want to migrate over to one main library (plus some plugin libraries, like Plotly)? [Here](https://airbnb.io/visx/gallery)'s an option.

Reply:
Yeah, it seems that visx has the benefit of small bundle sizes while something like [recharts](https://recharts.org/en-US/) definitely has the most adoption so far. I'll look more into what options are available but I like visx as an option.

Reply:
Recharts looks more polished out the box, but visx seems more customizable in the long run. Totally up to you!